buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkTextStyles.kt

```diff
@@ -59,19 +69,15 @@ private fun parseTextStyles(source: Map<String, Any>): BpkTextStyles {
   fun String.trimReference(): String =
     removePrefix("{!").removeSuffix("}")
 
-  fun String.toSemanticName() =
-    CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, this.replace('-', '_'))
-
-  fun String.wrapIfDigits() = if (first().isDigit()) { "`$this`" } else this
-
-  fun List<Map.Entry<String, Map<String, String>>>.findTokenName(type: String): String? {
+  fun List<Map.Entry<String, Map<String, String>>>.findToken(type: String): Token? {
```

Comment:
also here had to change the parsing and remove the case conversion/wrapping logic and move that into the formatter later on